guides/ipynb/keras_cv/cut_mix_mix_up_and_rand_augment.ipynb

```diff
@@ -225,7 +225,7 @@
     "\n",
     "You can read more about these\n",
     "parameters in the\n",
-    "[`RandAugment` API documentation](/api/keras_cv/layers/preprocessing/rand_augment/).\n",
+    "[`RandAugment` API documentation](https://keras.io/api/keras_cv/layers/preprocessing/rand_augment/).\n",
```

Comment:
That's the same link, no?

Reply:
I believe the relative link will break in colab.